lib/darwin/index.js

```diff
@@ -3,7 +3,10 @@ const exec = require('child_process').exec
 const temp = require('temp')
 const fs = require('fs')
 
-function darwinSnapshot () {
+function darwinSnapshot (windowTitle = "") {
+  if(windowTitle)
+    console.log('Window title not supported for Darwin OS; ignored ... ');
```

Comment:
Throwing a error here would be preferred so the error can be picked up programmatically.